### ISSUE TITLE
Markdown to HTML in circular heatmap views

### ISSUE THREAD

**Issue by cosad3s** [devsecopsmaturitymodel/DevSecOps-MaturityModel#342]
The markdown is correctly interpreted in Activity Description view.

*Example:* [https://dsomm.owasp.org/activity-description?uuid=47419324-e263-415b-815d-e7161b6b905e&dimension=Culture%20and%20Organization&subDimension=Design&level=1&activityName=Conduction%20of%20simple%20threat%20modeling%20on%20technical%20level](https://dsomm.owasp.org/activity-description?uuid=47419324-e263-415b-815d-e7161b6b905e&dimension=Culture%20and%20Organization&subDimension=Design&level=1&activityName=Conduction%20of%20simple%20threat%20modeling%20on%20technical%20level)

![image](https://github.com/user-attachments/assets/27ad38a1-884e-4363-ac47-e681ce53c38a)

But in Circular Heatmap, the content are in plain-text. Cannot be read if some styles are applied.
*Example for the same topic than above:* 
![image](https://github.com/user-attachments/assets/cacd5bda-2b19-4719-8766-4118c9049356)

Could it be possible to have a nice Markdown to HTML view also in circular heatmap ?

**Comment by wurstbrot** [comment]
Thank you for submitting the issue.

@0x41head @ptechofficial , ideas?

**Comment by 0x41head** [comment]
looking into it

**Comment by vbakke** [comment]
I don't think this quite solved, @0x41head. It is only fixed for Description, but not for Risk and Measurement (which also contain markdown, just not as often).

Also, I think you'd be better off solving it as done in `activity-description.component.ts` at load time, using `defineStringValues()`. Then we don't need to mock about in angular. :)
